.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
this along with the 'engined' in the `package.json` will make npm complain if you try to `nop install` with the wrong npm version

Reply:
nice - good call